.github/workflows/create-docs-issues.yml

```diff
@@ -0,0 +1,53 @@
+name: Create documentation ticket
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  create-docs-ticket:
+    # Only on merged PRs that contain the DocumentationNeeded flag
+    if: github.event.pull_request.merged && contains(github.event.pull_request.labels.*.name, 'DocumentationNeeded')
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Create Issues
+        id: create-issues
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.DOCS_ISSUES_PAT }}
+          script: |
+            const ISSUE_REPO_NAME = 'deephaven.io';
+            const issueTypes = ['how-to', 'conceptual', 'reference', 'blog'];
+
+            const body = `_This issue was auto-generated_\n\n` +
+              `PR: ${context.payload.pull_request.html_url}\n` +
+              `Author: ${context.payload.pull_request.user.login}\n\n` +
+              '## Original PR Body\n\n' +
+              context.payload.pull_request.body;
+
+            const promises = issueTypes.map(type => github.rest.issues.create({
+              owner: context.repo.owner,
+              repo: ISSUE_REPO_NAME,
+              title: `${type.toUpperCase()}: ${context.payload.pull_request.title}`,
+              body,
+              labels: [type, 'documentation', 'triage', 'autogenerated']
+            }));
+
+            const issues = await Promise.allSettled(promises);
```

Comment:
What happens if there's an error creating the tickets - action fails and we see the error there?

Reply:
Nevermind - it'll appear in the comment. Cool.